rllib/env/single_agent_env_runner.py

```diff
@@ -623,40 +559,6 @@ def stop(self):
         # Close our env object via gymnasium's API.
         self.env.close()
 
-    # TODO (sven): Replace by default "to-env" connector.
```

Comment:
Not necessary anymore here in EnvRunner.

This is default ModuleToEnv connector behavior now.